src/main/java/core/basesyntax/impl/ConvertingFileToFruitTransactionImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.ConvertingFileToFruitTransaction;
+
+import java.util.List;
+
+public class ConvertingFileToFruitTransactionImpl implements ConvertingFileToFruitTransaction {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convert(List<String> lines) {
+        lines.remove(0);//remove line with (type,fruit,quantity)
+        return lines.stream()
+                .map(n -> toFruitTransaction(n))
+                .toList();
+    }
+
+    private FruitTransaction toFruitTransaction(String line) {
+        String[] strings = line.split(",");
+        return new FruitTransaction()
+                .setOperation(FruitTransaction.Operation.fromCode(strings[OPERATION_INDEX]))
+                .setFruit(strings[FRUIT_INDEX])
+                .setQuantity(Integer.parseInt(strings[AMOUNT_INDEX]));
```

Comment:
Consider adding error handling for `Integer.parseInt(strings[AMOUNT_INDEX])`. If the amount is not a valid integer, this will throw a `NumberFormatException`. You should catch this exception and throw a more informative `RuntimeException`.